lib/screens/login/login.dart

```diff
@@ -39,40 +46,182 @@ class LoginScreen extends StatelessWidget {
+                              S.assetEcellLogoWhite,
+                              width: width * 0.25 * heightFactor,
+                            ),
+                          ),
+                          Container(
+                            alignment: Alignment.centerLeft,
+                            padding: EdgeInsets.only(left: D.horizontalPadding, top: 20),
+                            child: Text(
+                              "Welcome",
+                              style: TextStyle(
+                                  fontSize: 35 * heightFactor, fontWeight: FontWeight.w600),
+                            ),
+                          ),
+                          //Text Greeting
+                          Container(
+                            alignment: Alignment.centerLeft,
+                            padding: EdgeInsets.only(left: D.horizontalPadding, top: 5),
+                            child: RichText(
+                              text: TextSpan(
+                                text: "Let's ",
+                                children: [
+                                  TextSpan(
+                                      text: "Sign ",
+                                      style: TextStyle(color: C.primaryHighlightedColor)),
+                                  TextSpan(
+                                      text: "you in",
+                                      style: TextStyle(color: C.primaryUnHighlightedColor)),
+                                ],
+                                style: TextStyle(fontSize: 25 * heightFactor),
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                      flex: 4),
+                  //Form
+
+                  Flexible(
+                    child: Form(
+                        key: _formKey,
+                        child: Padding(
+                          padding: EdgeInsets.symmetric(horizontal: D.horizontalPadding),
+                          child: Column(
+                            children: [
+                              SizedBox(height: 10 * heightFactor),
+                              EmailField(emailController),
+                              SizedBox(height: 30 * heightFactor),
+                              PasswordField(passwordController),
+                              SizedBox(height: 10 * heightFactor),
+                            ],
+                          ),
+                        )),
+                    flex: 4,
+                  ),
+                  //Redirect to Forgot Password
+
+                  Expanded(
+                      flex: 1,
+                      child: Container(
+                        padding: EdgeInsets.only(right: D.horizontalPadding),
+                        alignment: Alignment.topRight,
+                        child: RichText(
+                          text: TextSpan(text: "Forgot Password?"),
```

Comment:
1. RichText is used when you wish to display a line(s) of text with different text styles. Since we have just one line with only one style, its not needed here. It should just be `Text`.
2. Color should be `C.secondaryColor`. Refer designs.
3. Font size should be bigger. Refer designs.
4. Surround with a `GestureDetector` with a TODO in `onTap`.